### PR TITLE
Bug fix : Container widget background color is not consistent with the dark theme

### LINKED ISSUE #2649: **Container** widget does not adapt to the dark theme
### What is the expected behaviour?
**Container** widget should adapt to the dark theme.

### What is the current behaviour?
**Container** widget does not adapt to the dark theme

### How to reproduce the issue?

1. Add Container widget to the canvas from widget manager
2. Toggle to the dark mode

### Screenshots or S
Screenshot 2022-03-28 at 2 43 27 PM
creencast

### PR DESCRIPTION
**Pull request by manishkushare**
Resolves #2649

### CODE REVIEW

**Comment by shubh22** [comment]
Task linked: [CU-2r6cdzw Container widget does not adapt to the dark theme](https://app.clickup.com/t/2r6cdzw)

**Review by arpitnath — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/67645175/187246673-57c4cddb-f486-4fc8-a507-b1874e2df89c.mov

@manishkushare If we pick white (#FFFFFF) then the dark mode is not handled.

File: frontend/src/Editor/Components/Container.jsx [hunk 1]
```diff
@@ -2,9 +2,20 @@ import React, { useRef } from 'react';
...
+  component,
+  width,
+  height,
+  containerProps,
+  removeComponent,
+  styles,
+  darkMode,
+}) {
+  const { visibility, disabledState, borderRadius } = styles;
+  console.log(styles.backgroundColor);
```
Comment (arpitnath) [inline frontend/src/Editor/Components/Container.jsx:16]: Can we remove this 😅

**Review by arpitnath — APPROVED** [review]
LGTM!

File: frontend/src/Editor/Components/Container.jsx (hunk 1 above)
Comment (manishkushare) [reply frontend/src/Editor/Components/Container.jsx]: @arpitnath , removed console.log

**Review by arpitnath — APPROVED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**